.github/workflows/main.yml

```diff
@@ -21,6 +21,10 @@ jobs:
       - name: install and tests
         run: |
           yarn install
-          yarn build
+          yarn test
+          yarn size
         env:
           CI: true
+      - uses: preactjs/compressed-size-action@v2
+        with:
+          repo-token: '${{ secrets.GITHUB_TOKEN }}'
```

Comment:
They have different purpose. SL is for developer, the action is for you as a reviewer.

I would rather keep size-limit because:
1. It's not impossible to measure exports size using the GH action
2. It's not possible to check the size during the development using the GH action

Reply:
Ok, so let's leave size-limit only. 
The problem of several similar solutions when their data is different you have to check both of them.

Reply:
Also if we're going to get rid of microbundle, size-limit is the only way to measure **minified** and gzipped size. The action is just gzips the files, do we won't be able to know the real bundle size if we drop both size limit and microbunlde